03-transactions.md

```diff
@@ -373,7 +445,9 @@ outputs) is 7140 satoshi. The final fee may be even higher if the
 
 ### Fee Payment
 
-Base commitment transaction fees are extracted from the funder's amount; if that amount is insufficient, the entire amount of the funder's output is used.
+Base commitment transaction fees and amounts for `to_local_anchor` and `to_remote_anchor` outputs are extracted from the funder's amount;
+Restrictions to the commitment tx output for the funder in relation to the
+channel reserve apply as described in BOLT 02.
```

Comment:
link to BOLT02 description?

Reply:
pre-existing, but fixed in the first commit of this pr

Reply:
not pre-existing i see now